README.md

```diff
@@ -1,13 +1,17 @@
 # 📦✨  wasm-pack
-> pack up the wasm and publish it to npm!
+> your favorite rust -> wasm workflow tool!
 
 [![Build Status](https://travis-ci.org/ashleygwilliams/wasm-pack.svg?branch=master)](https://travis-ci.org/ashleygwilliams/wasm-pack)
 [![Build status](https://ci.appveyor.com/api/projects/status/7jjuo5wewu9lyyfi?svg=true)](https://ci.appveyor.com/project/ashleygwilliams/wasm-pack)
 
-the goal of this project is to create a portable command line tool
-for publishing compiled wasm projects to the npm registry for the consumption
-of js devs using the npm CLI, yarn, or any other CLI tool that interfaces
-with the npm registry.
+this tool seeks to be a one-stop shop for building and working with rust-
+generated webassembly that you would like to interop with JavaScript, in the
+browser or with Node.js. `wasm-pack` helps you build and publish rust-generated
```

Comment:
rust-generated -> rust generated

Reply:
why not the hyphen? the hyphen feels correct here... as rust modifies how it is generated?